src/emc/kinematics/genhexkins.c

```diff
@@ -310,6 +384,14 @@ int kinematicsForward(const double * joints,
         return -1;
       }
       pmCartMag(&InvKinStrutVect, &InvKinStrutLength);
+
+      if (haldata->screw_lead != 0.0) {
```

Comment:
calculations appear to be identical to before-patch as long as screw_lead is 0 (this is good)